docker/Dockerfile-light

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.4
+
+# KOPS_GITISH: Modify to build at an explicit tag/gitish
+ARG KOPS_GITISH=HEAD
+
+# KUBECTL_SOURCE: Change to kubernetes-dev/ci for CI
+ARG KUBECTL_SOURCE=kubernetes-release/release
+
+# KUBECTL_TRACK: Currently latest from KUBECTL_SOURCE. Change to latest-1.3.txt, etc. if desired.
+ARG KUBECTL_TRACK=latest.txt
+
+ARG KUBECTL_ARCH=linux/amd64
+
+RUN apk add --no-cache --update curl jq && \
+    KOPS_URL=$(curl -s https://api.github.com/repos/kubernetes/kops/releases/latest | jq -r ".assets[] | select(.name == \"kops-linux-amd64\") | .browser_download_url") && \
```

Comment:
Yeah, this really impressed me!

I couldn't make it work with other "releases" though despite it [being documented](https://developer.github.com/v3/repos/releases/#get-a-single-release) on the GitHub API.
It'd be good if we could pin the tags,